docs/pages/guides/server-components.mdx

```diff
@@ -207,6 +207,98 @@ export default function Button({ title }) {
 - Server actions can only return serializable data.
 - Server Actions run on the server and are a good place to put logic that should not be exposed to the client.
 
+### Rendering in Server Actions
+
+Server Actions in Expo Router can render React components on the server and stream back an **RSC payload** (a custom JSON-like format that's maintained by the React team) for rendering on the client. This is similar to server-side rendering (SSR) on the web,
+
+For example, the following server action will render some text:
+
+```tsx components/server-actions.tsx
+'use server';
+
+// Optional: Import "server-only" for sanity.
+import 'server-only';
+
+import { View, Image, Text } from 'react-native';
+
+export async function renderProfile({
+  username,
+  accessToken,
+}: {
+  username: string;
+  accessToken: string;
+}) {
+  // NOTE: Rate limits, GDPR, and other server-side operations can be done here.
+
+  // Fetch some data securely from an API.
+  const { name, image } = await fetch(`https://api.example.com/profile/${username}`, {
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+      // Use secret environment variables securely as this code will live on the server.
+      // The EXPO_PUBLIC_ prefix is not required here.
+      'X-Secret': process.env.SECRET,
+    },
+  }).then(res => res.json());
+
+  // Render
+  return (
+    <View>
+      <Image source={{ uri: image }} />
+      <Text>{name}</Text>
+    </View>
+  );
+}
+```
+
+This server action can be invoked from a client component and the contents will be streamed back to the client:
```

Comment:
```suggestion
This Server Action can be invoked from a Client Component and the contents will be streamed back to the client:
```

Since in the last PR we decided to use the case as suggested above.